x/proof/keeper/proof_validation.go

```diff
@@ -258,21 +331,20 @@ func (k Keeper) validateClosestPath(
 	return nil
 }
 
-// queryAndValidateClaimForProof ensures that a claim corresponding to the given
-// proof's session exists & has a matching supplier operator address and session header,
-// it then returns the corresponding claim if the validation is successful.
-func (k Keeper) queryAndValidateClaimForProof(
+// validateClaimForProof ensures that a claim corresponding to the given proof's
```

Comment:
There is no given proof. I feel like this should justbe called "validateSessionClaim"